jwst/coron/klip.py

```diff
@@ -88,11 +92,25 @@ def klip(target_model, refs_model, truncate):
     return output_target, output_psf
 
 
-def KarhunenLoeveTransform(m, normalize=False):
+def karhunen_loeve_transform(m, normalize=False):
     """
-    Returns Karhunen-Loeve Transform of the input, eigenvalues, and
-    a matrix of eigenvectors.
+    Calculate Karhunen-Loeve Transform of the input.
 
+    Parameters
+    ----------
+    m : numpy ndarray
+        The array of flattened, background subtracted reference arrays
+    normalize : bool, default=False
+        Normalize the returned transform?
+
+    Returns
+    -------
+    klvect : numpy ndarray
```

Comment:
Similar changes should be consistent throughout in this PR at least. Same comment applies to all similar occurrences. 
```suggestion
    klvect : numpy.ndarray
```

Reply:
Fixed in the latest commit